websites/J/JAVX/presence.ts

```diff
@@ -0,0 +1,143 @@
+const presence = new Presence({
+    clientId: "830476272978362408",
+    injectOnComplete: true,
+    appMode: true
+  }),
+  strings = presence.getStrings({
+    playing: "general.playing",
+    paused: "general.paused"
+  });
+
+let video = { playback: false, currentTime: 0, duration: 0, paused: false };
+
+presence.on(
+  "iFrameData",
+  (data: {
+    playback: boolean;
+    currentTime: number;
+    duration: number;
+    paused: boolean;
+  }) => {
+    video = data;
+  }
+);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "javx",
+      startTimestamp: Math.floor(Date.now() / 1000)
+    },
+    { pathname, search } = document.location;
+
+  if (search.startsWith("?s")) {
+    const s = new URLSearchParams(search).get("s");
+
+    if (s) {
+      presenceData.details = "Đang tìm phim với từ khoá:";
+      presenceData.state = encodeURI(s);
+    } else {
+      presenceData.details = "Đang tìm phim";
+    }
+  } else if (pathname === "/") {
+    presenceData.details = "Đang ở trang chủ";
+  } else if (pathname.startsWith("/sign-up")) {
+    presenceData.details = "Đang đăng ký tài khoản";
+  } else if (pathname.startsWith("/sign-in")) {
+    presenceData.details = "Đang đăng nhập";
+  } else if (pathname.startsWith("/profile")) {
+    presenceData.details = "Đang xem hồ sơ cá nhân";
+  } else if (pathname.startsWith("/favorite")) {
+    presenceData.details = "Đang xem danh sách các phim đã lưu";
+  } else if (pathname.startsWith("/censored")) {
+    presenceData.details = "Đang ở trang JAV có che";
+  } else if (pathname.startsWith("/uncensored")) {
+    presenceData.details = "Đang ở trang JAV không che";
+  } else if (pathname.startsWith("/porn")) {
+    presenceData.details = "Đang ở trang Porn châu Âu";
+  } else if (pathname.startsWith("/actresses")) {
+    presenceData.details = "Đang xem danh sách JAV Idols (diễn viễn JAV)";
+  } else if (pathname.startsWith("/studios")) {
+    presenceData.details = "Đang xem danh sách JAV Studios (hãng sản xuất JAV)";
+  } else if (pathname.startsWith("/years")) {
+    presenceData.details = "Đang xem danh sách các năm phát hành JAV";
```

Comment:
You don't need to have brackets here since there is only one line of code in them.